components/sidebar/sidebar_channel/sidebar_channel.tsx

```diff
@@ -173,11 +189,13 @@ export default class SidebarChannel extends React.PureComponent<Props, State> {
                     index={channelIndex}
                 >
                     {(provided, snapshot) => {
+                        const {handleTransitionEnd, restOfDraggableProps} = this.wrapTransitionEndHandler(provided.draggableProps);
+
```

Comment:
draggableProps also injects onTransitionEnd, so we need to extract it from the props via factory function